### PR TITLE
Use `promote.is_external` to test for external promotions

### PR DESCRIPTION
**Pull request by dwick**
👓 @zeantsoi 

related to https://github.com/reddit/reddit-public/pull/5030

### CODE REVIEW

File: reddit_adzerk/adzerkpromote.py [hunk 1]
```diff
@@ -153,7 +153,9 @@ def update_campaign(link, az_advertiser=None, triggered_by=None):
         'SalespersonId': g.az_selfserve_salesperson_id,
         'IsDeleted': False, # deleting an adzerk object will make it
                             # unretrievable, so just set it inactive
-        'IsActive': promote.is_accepted(link) and not link._deleted,
+        'IsActive': (promote.is_accepted(link) or
+                     promote.is_external(link)) and
+                     not link._deleted,
```
Comment (zeantsoi) [inline reddit_adzerk/adzerkpromote.py]: Just checking: does the entire condition need to be wrapped in parentheses because it's multiline?

**Comment by zeantsoi** [comment]
:nail_care:

File: reddit_adzerk/adzerkpromote.py (hunk 1 above)
Comment (dwick) [inline reddit_adzerk/adzerkpromote.py]: python!

**Comment by dwick** [comment]
💇

**Comment by zeantsoi** [comment]
:fish:

**Pull request merged**